token-holders/participate-in-governance.md

```diff
@@ -0,0 +1,17 @@
+# Participate in governance
+
+The Pyth Network is governed by Pyth token-holders. They have the ultimate authority over the future of the protocol. 
+
+Any wallet that owns Pyth tokens can participate in governance. Doing so requires locking Pyth tokens in the staking contract. 
+
+Governance involves interacting with two programs on the Solana blockchain. 
```

Comment:
Suggest adding the program addresses *after* the bullet points below pointing the user to the two steps on how to vote.

Reply:
Done